src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaInspector.tsx

```diff
@@ -401,8 +421,9 @@ const SchemaInspector = ((props: ISchemaInspectorProps) => {
     <div>
       <p className={classes.name}>{getTranslation('name', props.language)}</p>
       <TextField
-        id={`${getDomFriendlyID(selectedId ?? '')}-name`}
+        id='selectedItemName'
```

Comment:
How are you guarding this id from becoming a duplicate as the schema grows?
If we later add the htmlFor attributes on labels, they will not uniquely point to a single id...

Reply:
There is only ever one instnace of this textfield, its the name field in the inspector window